manifests/init.pp

```diff
@@ -209,6 +210,10 @@
     version    => '1.0',
   }
 
+  if $use_hiera {
```

Comment:
@fbretel And here, it would simply become `create_resources('sonarqube::plugin', $plugins)`.